src/utils/currencies.ts

```diff
@@ -4,27 +4,32 @@ import { getTokenAmountString } from "@hermeznetwork/hermezjs/src/utils";
 import { MAX_TOKEN_DECIMALS } from "src/constants";
 import { FiatExchangeRates, Token } from "src/domain/hermez";
 
+export type Currency = {
+  code: string;
+  symbol: string;
+};
+
 const CurrencySymbol = {
   USD: {
     symbol: "$",
     code: "USD",
-  },
+  } as Currency,
```

Comment:
Instead of doing this `as Currency` all the time is better to type the const like this:

```ts
const CurrencySymbol: Record<string, Currency> = // ...
```